startup/routes.js

```diff
@@ -16,7 +16,7 @@ module.exports = function (app) {
   app.use(express.json())
 
   app.use('/api', limit)
-  app.use('/api/v1/users', example)
+  app.use('/api/user', authRoute)
```

Comment:
user will be used for user routes
change path to /api/auth, and as second argument set just auth not authRoute